modules/ROOT/pages/index.adoc

```diff
@@ -25,51 +25,53 @@ Discover Anypoint Platform APIs on https://anypoint.mulesoft.com/exchange/portal
 
 == Design and Build API-based Integrations
 
-Anypoint Studio is an Eclipse-based IDE that helps you create integrations. Design Center, a web-based tool, helps you create API specifications, the foundation for data integrations.
+Anypoint Studio is an Eclipse-based IDE that enables you to create integrations. Design Center, a web-based tool, enables you to create API specifications, which are the foundation for data integrations.
 
+[%header%autowidth.spread]
 |===
-| **Creating Integrations** | **Creating API Specifications**
+| Creating Integrations | Creating API Specifications
 | xref:studio::index.adoc[Building Integrations with Anypoint Studio] | xref:studio::import-api-specification-exchange.adoc[Importing APIs from Exchange to Studio]
 | xref:design-center::about-designing-a-mule-application.adoc[Building Integrations in Design Center]  | xref:design-center::design-create-publish-api-specs.adoc[Creating API specifications in Design Center]
 |===
 
-Studio 7 includes xref:mule-runtime::index.adoc[Mule Runtime (Mule) 4], the runtime environment where you deploy Mule apps and APIs.
+Anypoint Studio 7.0 and later includes xref:mule-runtime::index.adoc[Mule Runtime (Mule) 4], the runtime environment where you deploy Mule apps and APIs.
 Earlier versions of Studio include xref:3.9@mule-runtime::index.adoc[Mule 3].
 
+[%header%autowidth.spread]
 |===
-| **Mule 3**    | **Mule 4**
+| Mule 3    | Mule 4
 | xref:3.9@mule-runtime::anypoint-connectors.adoc[Connectors for Mule 3] | xref:connectors::index.adoc[Connectors for Mule 4]
 | xref:3.9@mule-runtime::dataweave.adoc[DataWeave 1 for Mule 3]  | xref:mule-runtime::dataweave.adoc[DataWeave 2 for Mule 4]
 | xref:3.x@apikit::index.adoc[APIkit 3] | xref:4.x@apikit::overview-4.adoc[APIkit 4]
-| xref:3.9@connector-devkit::index.adoc[Connector DevKit] | xref:1.1@mule-sdk::index.adoc[Mule SDKs]
+| xref:3.9@connector-devkit::index.adoc[Connector DevKit] | xref:1.1@mule-sdk::index.adoc[Mule SDK]
 |===
 
 == Test Integrations
 
-Test the integrations you create before you deploy them.
+It is best practice to test the integrations you create before you deploy them:
```

Comment:
Try to avoid beginning sentences with the "false subjects" "It is" and "There are":
```suggestion
Best practices dictate that you test the integrations you create before you deploy them:
```
...or some alternative to "dictate that you" that I cannot come up with right now.... ;-)